pkgs/development/python-modules/robotstatuschecker/default.nix

```diff
@@ -8,7 +8,7 @@ buildPythonPackage rec {
   src = fetchFromGitHub {
     owner = "robotframework";
     repo = "statuschecker";
-    rev = version;
+    rev = "refs/tags/v${version}";
```

Comment:
Isn't that basically just `v${version}`? Or is that ambigious in that case?

Reply:
not sure, I just used the bigger hammer

Reply:
Yes, it is and they have no conflicting branch with the same name.